js/viewer/annotation/annotation-svg.js

```diff
@@ -528,7 +499,7 @@ function AnnotationSVG(parent, id, imgWidth, imgHeight, scale, ignoreReferencePo
                     if(id !== "undefined"){
                         group = this.groups.hasOwnProperty(id) ? this.groups[id] : this.createAnnotationGroup(id, anatomy);
                         annotation = group.addAnnotation(node.nodeName);
-                        annotation.setAttributesBySVG(node);
+                        annotation.setAttributesByJSON(this.getNodeAttributes(node.attributes));
```

Comment:
Instead of passing `node.attributes`, pass `node`.

Reply:
Why should `node` be passed instead of `node.attribute` ?

Reply:
It's called `getNodeAttributes`, so you'd expect to get it from the node. Also, your function implementation is just confusing and not easy to look at as it's assuming the input. If your input is an `Element` typed object it's more straightforward.

Reply:
yeah that makes sense. I will correct that. Are you talking about the implementation of the `getNodeAttributes` function. If so then that is because the `node.attributes` is a JS `NamedNodeMap` (https://developer.mozilla.org/en-US/docs/Web/API/NamedNodeMap), and therefore i had to make the function so that I can get the desired output.